systemtests/tests/testfind/testrunner

```diff
@@ -2,10 +2,7 @@
 set -e
 set -o pipefail
 set -u
-#
-# Run a simple backup
-#   then restore it.
-#
+
```

Comment:
```suggestion
#
# Check whether testfind returns correct results
#  and handles errors as we expect.
#
```
Not sure if this is needed since this test does not do much, but its always nice to have descriptions

Reply:
no need, the test is already small and self-explanatory